classes/Project/Project.mjs

```diff
@@ -141,6 +141,19 @@ export default class Project {
     }
   }
 
+  async updateMetadata(newMetadata) {
+    this.data.metadata = newMetadata
+    return await this.update()
+  }
+
+  async update() {
+    return await database.update({ ...this.data, type: "Project" })
```

Comment:
It is probably worth making the change to `update()` that does not require `@type: Project` https://github.com/CenterForDigitalHumanities/TPEN-services/blob/9314bd0ffee370a2260b3a7abee5f896716d1a8d/database/mongo/controller.mjs#L217 The change was already made to `save()`

Reply:
Clearly recognize the two layers here.  

The class Project has a `save()` and `update()` that treat the need for `"Project"` a little differently.  At a minimum reconcile this.

The mongo controller has a `save()` and `update()` that treat the need for "Project" in a significantly different way.  Determine if it is within the scope of this PR to reconcile the difference in the mongo controller as well.